Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -32,8 +50,8 @@ await useCase.Execute(userUrn)
             .WithOpenApi(operation =>
             {
                 operation.OperationId = "ListOrganisations";
-                operation.Description = "Get a list of organisations.";
-                operation.Summary = "Get a list of organisations.";
+                operation.Description = "Get a list of organisations for a particular userUrn.";
+                operation.Summary = "Get a list of organisations for a particular userUrn.";
```

Comment:
Placing the `userUrn` here was a mistake. It was to accommodate the organisation selection screen, which was wrongly using this endpoint. It was recently fixed and the organisation selection screen no longer uses this endpoint. You should be using it instead of introducing the new `/admin/organisations`.

Reply:
It's still the same (requires a userUrn) ... `FindByUserUrn`
<img width="699" alt="image" src="https://github.com/user-attachments/assets/709a32b8-0451-4bb0-b34b-d0bfea318782">

I cannot use this endpoint